packages/get-platform/src/__tests__/getPlatform.test.ts

```diff
@@ -0,0 +1,140 @@
+import { getPlatformInternal as getPlatform } from '../getPlatform'
+import { jestConsoleContext, jestContext } from '../test-utils'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+describe('getPlatform', () => {
+  describe('linux', () => {
+    const platform = 'linux'
+
+    // test name convention: <originalDistro> (<familyDistro>), <arch> (<uname -m>), openssl-<libssl>
+
+    it('debian (debian), amd64 (x86_64), openssl-1.1.x', () => {
+      expect(
+        getPlatform({
+          platform,
+          libssl: '1.1.x',
+          arch: 'x64',
+          archFromUname: 'x86_64',
+          familyDistro: 'debian',
+          originalDistro: 'debian',
+          targetDistro: 'debian',
+        }),
+      ).toBe('debian-openssl-1.1.x')
+      expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.warn'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    })
```

Comment:
We now have warning/log/error snapshot testing for `getPlatform`, like Jan wanted